IzvorniKod/backend/src/main/java/com/educhat/backend/services/UserService.java

```diff
@@ -44,4 +44,8 @@ public User authenticateUser(String usernameOrEmail, String rawPassword) {
         }
         return user;
     }
+
+    public User getUserById(Long userId) {
+        return userRepository.findById(userId).orElseThrow(() -> new RuntimeException("User not found"));
```

Comment:
Can you make a custom exception for this?

You can see in exceptions folder how its done. Create an exception and then handle it with GlobalExceptionHandler

Reply:
napravljeno, ako treba jos nesto javi